NanoServerPackage/Test/NanoServerPackage.Save.Tests.ps1

```diff
@@ -59,6 +59,31 @@ Describe "Save-WindowsPackage Stand-Alone" {
         Remove-Item $pathToSave\*.cab
     }
 
+    It "Save-NanoServerPackage Name, Path, Force" {
+        $name = $names | Get-Random
+        $version = $requiredVersion
+
+        $SavePath = "$TestDrive\SaveFolder"
+
+        $results = @()
+        $results += (Save-NanoServerPackage -Name $name -Path $SavePath -Force)
+
+        $results.count | should be 1
+        $results[0].name | should be $name
+        $results[0].version | should be $version
+        $results[0].culture | should be "en-us"
```

Comment:
removed

---
In reply to: [82315366](https://github.com/OneGet/NanoServerPackage/pull/6#discussion_r82315366) [](ancestors = 82315366)